lib/modules.nix

```diff
@@ -1036,6 +1036,17 @@ let
   defaultOrderPriority = 1000;
   mkAfter = mkOrder 1500;
 
+  /*
+    For use with options of type `attrsOf *`, `lazyAttrsOf *`, or `listOf *`.
+    Accepts a value that is merged with every element defined elsewhere. May
+    also accept a function that takes the attribute name or index (1-based),
+    the result of which is used as above.
+  */
+  mkForAllValues = content:
```

Comment:
This thread preallocated for bikeshedding the name of this function.

`forEach` is used elsewhere in the libraries, but there it means ‘flipped `map`’, and this function is not a map at all (if it is given a function, the argument provided to that function is key, not value). So I went with `ForAll` to draw the distinction.

`Values` splits the difference between `Elems` for lists and `Attrs` for attrsets, but I'm not wedded to it.

Reply:
`Items` might be better. Everything except an attribute is already value in the expression language.

`types.nix` currently has `elemType` in `nestedTypes`, which I think was a mistake because `item` is a word and `elem` is not. (Also `elem` is a _function_ in Haskell, fwiw)

Reply:
Changed to `Items`.